examples/ota-requestor-app/linux/main.cpp

```diff
@@ -195,40 +203,39 @@ int main(int argc, char * argv[])
     SetDeviceAttestationCredentialsProvider(chip::Credentials::Examples::GetExampleDACProvider());
 
     // Initialize and interconnect the Requestor and Image Processor objects -- START
-    // Initialize the instance of the main Requestor Class
-    OTARequestor * requestorCore = new OTARequestor;
-    SetRequestorInstance(requestorCore);
-
-    // Initialize an instance of the Requestor Driver
-    LinuxOTARequestorDriver * requestorUser = new LinuxOTARequestorDriver;
+    SetRequestorInstance(&requestorCore);
 
     // Connect the Requestor and Requestor Driver objects
-    requestorCore->SetOtaRequestorDriver(requestorUser);
+    requestorCore.SetOtaRequestorDriver(&requestorUser);
 
-    // Initialize  the Downloader object
-    OTADownloader * downloaderCore = new OTADownloader;
-    // TODO: enable    SetDownloaderInstance(downloaderCore);
-
-    // Initialize the Image Processor object
-    LinuxOTAImageProcessor * downloaderUser = new LinuxOTAImageProcessor;
+    // WARNING: this is probably not realistic to know such details of the image or to even have an OTADownloader instantiated at
+    // the beginning of program execution. We're using hardcoded values here for now since this is a reference application.
+    // TODO: instatiate and initialize these values when QueryImageResponse tells us an image is available
```

Comment:
Yeah, imageFile should be what's set in the URI. I can make this change as part of my CASESessionManager PR.

Reply:
Sure, just want to be clear in the code what's missing